song-server/src/main/resources/schemas/analysis/analysisTypeRegistration.json

```diff
@@ -10,9 +10,25 @@
             "const": "object"
         },
         "definitions" : {
-          "type" : "object"
+          "type" : "object",
+            "properties" : {
+                "file" : {
+                    "type": "object",
+                    "properties": {
+                        "fileType": {
+                            "oneOf": [
+                                { "$ref": "classpath:/schemas/analysis/analysisBase.json#/definitions/file/fileType"},
+                                {
+                                    "type": "string",
+                                    "enum": ["VCF", "BAM", "OTHER_CUSTOM_TYPE"]
+                                }
+                            ]
+                        }
+                    }
+                }
+            }
```

Comment:
It looks like, you have provided these `enum` values as an example... This schema is used to validate the content submitted when a user is defining a dynamic schema, and this would allow them to provide a schema with `file.fileType: "OTHER_CUSTOM_TYPE"`. This is definitely not what we want.

What we want is the user to provide additional restrictions to the `classpath:/schemas/analysis/analysisBase.json#/definitions/file/fileType` definition, for instance providing a custom enum value. This is so that when a new analysis is submitted those validation rules are applied.